static/js/metrics-explorer.js

```diff
@@ -153,6 +153,23 @@ function getUrlParameter(name) {
     let results = regex.exec(location.search);
     return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
 }
+// Updates saved Metrics Url on changing in metrics Explorer
+function updateMetricsQueryParamsInUrl() {
+    var currentPage = window.location.pathname;
+    const urlParams = new URLSearchParams(window.location.search);
+
+    if (currentPage.includes('metrics-explorer.html') && urlParams.has('queryString')) {
```

Comment:
We don't need to create the URL only when opening a saved link; it should always be present. So, you can remove the check for `urlParams.has('queryString')`.